src/client/client.go

```diff
@@ -245,6 +260,9 @@ func WithAdditionalPachdCert() Option {
 func getCertOptionsFromEnv() ([]Option, error) {
 	var options []Option
 	if certPaths, ok := os.LookupEnv("PACH_CA_CERTS"); ok {
+		if pachdAddress, ok := os.LookupEnv("PACHD_ADDRESS"); !ok || !strings.HasPrefix(pachdAddress, "grpcs") {
```

Comment:
Nit: should we put "grpcs" in a constant at the top of the file?

Reply:
I decided to use "grpcs://" because even the instances when we just use "grpcs" should also be using "grpcs://